.changes/unreleased/NEW FEATURES-20250107-125354.yaml

```
@@ -0,0 +1,5 @@
+kind: NEW FEATURES
+body: Add write-only attributes to resources. Providers can specify that certain attributes are write-only, they can not be referenced and are not presisted in state. You can use ephemeral values in write-only attributes.
```

```suggestion
body: Add write-only attributes to resources. Providers can specify that certain attributes are write-only. They are not persisted in state. You can use ephemeral values in write-only attributes.
```

Fixed a typo. Also I don't think we actually ended up adding the validation for references to write-only attributes. I thought it all falls under https://github.com/hashicorp/terraform/pull/36171 as `#wontfix` but I'm happy to be proven wrong.

---

Ah okay, I was not aware of that (or I forgot over the holidays) :)